This patch for speech-tools for Debian GNU/Linux,
is to allow compiling with build hardening enabled.
The build hardener requires an explicit form for fprintf calls.

diff --git a/siod/siodeditline.c b/siod/siodeditline.c
index 403ae01..cb8e1f2 100644
--- a/siod/siodeditline.c
+++ b/siod/siodeditline.c
@@ -151,7 +151,7 @@ static STATUS siod_manual()
     symbol = el_current_sym();
     infostring = siod_manual_sym(symbol);
     putc('\n',stderr);
-    fprintf(stderr,infostring);
+    fprintf(stderr,"%s",infostring);
     fflush(stderr);
     putc('\n',stderr);
     fflush(stderr);
diff --git a/utils/cmd_line.cc b/utils/cmd_line.cc
index 37f4e1d..24c6f50 100644
--- a/utils/cmd_line.cc
+++ b/utils/cmd_line.cc
@@ -460,7 +460,7 @@ static void output_sgml_synopsis(char **argv, const EST_String &usage)
     fprintf(stdout,"<cmdsynopsis><command>%s</command>", 
 	    (const char *)full.filename());
 
-    fprintf(stdout, (const char *)ts.get_upto_eoln().string());
+    fprintf(stdout, "%s", (const char *)ts.get_upto_eoln().string());
 
     while (!ts.eof())
     {
